kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -132,7 +132,7 @@ packagesMarketplaceInstallConfirmMessage=Install deployment package from Eclipse
 packagesDropzoneMessage=Drop the package to install
 packagesMarketplaceInstallDpNotValid=File or link not valid
 packagesTooManyDpFiles=Too many files selected. Only one file is allowed at a time.
-packagesIntro=Manage the Deployment Packages installed on the system
+packagesIntro=Manage the Deployment Packages installed on the system. Due to the asynchronous startup of the bundles contained in a deployment package, a page reload may be required. 
```

Comment:
I wuold add some words highlightning one of the effect is that the services name will not be displayed in the service list section. So the refresh will solve the missing.